dotCMS/src/main/java/com/dotcms/security/secret/Param.java

```diff
@@ -0,0 +1,35 @@
+package com.dotcms.security.secret;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Param extends AbstractProperty<String>{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A class which only has private constructors should be final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4623512)